package.json

```diff
@@ -65,79 +65,64 @@
   "dependencies": {
     "@actions/core": "^1.2.4",
     "@actions/github": "^4.0.0",
-    "@babel/preset-react": "^7.9.4",
-    "@babel/runtime": "^7.9.2",
```

Comment:
We don't need this any more? Are you sure @ndelangen?

Reply:
Everything seems to work fine without them?

Reply:
Hmm, I'm taking specifically about `@babel/runtime`. I suspect it could break if you used the package in a project which itself didn't have a dependency on `@babel/runtime`, as `@babel/plugin-transform-runtime` is going to put down a bunch of `require('@babel/runtime')` calls into to the transpiled code.

Reply:
fixed!